cfy_manager/config.py

```diff
@@ -32,6 +34,7 @@
     DEFAULT_CONFIG_PATH,
     CLOUDIFY_USER,
     CLOUDIFY_HOME_DIR,
+    INITIAL_INSTALL_DIR
```

Comment:
,

Reply:
Just out of curiosity, why is a comma needed at the end of the last item?